specification/migrate/resource-manager/readme.md

```diff
@@ -20,11 +20,28 @@ batch:
   - tag: package-migrateengine-2022-05
 ```
 
+
```

Comment:
There is no `Basic Information` part for SDK generation. Please refer to template https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/samplefiles/README.md#basic-information to add it.